RELEASING.md

```diff
@@ -1,8 +1,72 @@
-### Overview
-
-The release process for SNAPSHOT versions is handled by the `after_success.sh` script, triggered after Travis CI successfully completes a non-PR build. A new SNAPSHOT artifact will be released to the snapshot repository as part of this script.
-
-### To push a release version
```

Comment:
I suggest we make this instruction into two sections:
1) One-time setup (setup account, keys, settings)
2) The actual release instructions

if users have done section 1) before they can just skip to 2)